rslib/src/scheduler/states/review.rs

```diff
@@ -272,6 +272,11 @@ mod test {
         ctx.interval_multiplier = 0.1;
         assert_eq!(state.passing_review_intervals(&ctx), (2, 3, 4));
         ctx.fuzz_factor = Some(0.99);
-        assert_eq!(state.passing_review_intervals(&ctx), (2, 3, 4));
+        assert_eq!(state.passing_review_intervals(&ctx), (3, 5, 7));
```

Comment:
This illustrates the implications of ensuring at least 1 day of fuzzing. If you find this too extreme, we could restrict that logic to the hard interval. The other intervals wouldn't get fuzzed then, but always be 1 above the hard interval, effectively getting fuzzed across 2 days as well.
So in this case, we would have (3, 4, 5).

Reply:
It does feel a bit aggressive - a card with an ease of 1.3x and a 1 day interval could end up with a 5x increase on Good. 3,4,5 sounds like it could work. One other idea would be to do the opposite - pin hard to 2 and allow the fuzz only in good/easy in this low interval case, so we'd end up with something like 2,4,5 in the 0.99 case. No strong feelings either way at this point - WDYT?

Reply:
To be honest, I don't like either solution very much. Not sure what'd be best study-wise, but it makes the code harder to read and the scheduler less transparent, because it couples fuzzing with the answer button.
A very simple, but not as comprehensive solution would be to increase the threshold above which we enforce fuzzing from 1 to 2. That would lead to (2, 4, 6) for a high fuzz factor.